profiles/common/modules/custom/tmgmt_poetry/README.md

```diff
@@ -155,20 +119,71 @@ DGCOMM (see 'Installations step 1').
 
 [Go to top](#table-of-content)
 
+### Configure the DGT-Connector:
+Once the module is enabled and the settings properly setup, CEM will proceed
+with the module's configuration.
+
+Edit the translator labelled "DGT Connector (auto created)".
+
+In order to do this, navigate to:
+
+
+    Configuration > Regional and Language > Translation management Translator
+
+or go to:
+
+
+    admin/config/regional/tmgmt_translator/manage/poetry_en 
+
+ - Translator settings: [x] Auto accept finished translations
+   - Check this if you don't want to review a translation before publishing it.
+
+ - Translator plugin:
+   - This is shown for information, please do not change it.
+
+ - DGT Connector plugin settings:
+   - You should see 'Main "poetry_service" variable is properly set.' if you
+   have correctly followed the steps above. Otherwise, get back and check what
+   you forgot!
+   - Counter: you do not need to fill this. The counter is auto generated.
+   - Requester code: must always be *WEB* [See DGT reference explanation]
+   (#dgt-reference-number).
+   - Organization responsible, Author & requester: check the example provided.
+   - Contact usernames: should be the *username* (you connect to the network,
+  ecas or the proxy with your username) of the persons in charge of the request.
+  This is important as only these persons can view translation details in the
+  web app.
+   - DGT contacts: The email address of your contact person at DGT. They will
+   receive feedback on the translations you want to comment. That information
+   should have been provided to you by DGCOMM.
+   - Remote languages mappings: Here you should map language codes that are not
```

Comment:
We should remove the "Here" from "**Here** you should map language codes..." and add a "the" between "by" and "DGT-Connector".